test/test_torch.py

```diff
@@ -1748,6 +1748,25 @@ def _test_neg(self, cast):
     def test_neg(self):
         self._test_neg(self, lambda t: t)
 
+    def test_bitwise_not(self):
+        res = 0xffff - torch.arange(127, dtype=torch.int8)
+        for t in (torch.BoolTensor, torch.LongTensor, torch.IntTensor, torch.ShortTensor, torch.CharTensor):
+            if t == torch.BoolTensor:
+                a = torch.BoolTensor([True, False])
+                this_res = torch.BoolTensor([False, True])
```

Comment:
nit: 'expectedRes' or similar instead of this_res?

Reply:
I changed to `expected_this`. I meant "result in this iteration".